flutter/shell/platform/tizen/flutter_tizen_view.cc

```diff
@@ -375,16 +396,7 @@ void FlutterTizenView::SendWindowMetrics(int32_t left,
                                          double pixel_ratio) {
   double computed_pixel_ratio = pixel_ratio;
   if (pixel_ratio == 0.0) {
-    // The scale factor is computed based on the display DPI and the current
-    // profile. A fixed DPI value (72) is used on TVs. See:
-    // https://docs.tizen.org/application/native/guides/ui/efl/multiple-screens
-#ifdef TV_PROFILE
-    double dpi = 72.0;
-#else
-    double dpi = static_cast<double>(tizen_view_->GetDpi());
-#endif
-    double scale_factor = dpi / 90.0 * kProfileFactor;
-    computed_pixel_ratio = std::max(scale_factor, 1.0);
+    computed_pixel_ratio = ComputePixelRatio(tizen_view_.get());
```

Comment:
The `computed_pixel_ratio` variable should not be necessary.
```suggestion
    pixel_ratio = ComputePixelRatio(tizen_view_.get());
```